src/contracts/core/AVSDirectory.sol

```diff
@@ -448,6 +575,28 @@ contract AVSDirectory is
         }
     }
 
+    function initializeAllocationDelay(
+        address operator,
+        uint32 delay
+    ) external {
+        require(
+            delegation.isOperator(operator),
+            "AVSDirectory.initializeAllocationDelay: operator not registered to EigenLayer yet"
+        );
+        require(
+            delegation.allocator(operator) == msg.sender,
```

Comment:
doesn't exist, no?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/691/commits/db66da58945da71f7f53940f6dbf656a7d25d61e

Reply:
not sure if we want to nuke from interface as well?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/691/commits/93ba8afce48fad4f4b2d0d357dc31e190482ce88